sideboard/lib/_websockets.py

```diff
@@ -4,7 +4,7 @@
 import json
 from copy import deepcopy
 from itertools import count
-from threading import RLock
+from threading import RLock, Event
```

Comment:
The changes in this file all related to "passthrough subscriptions" which MAGFest definitely doesn't care about at the moment.  If you're interested, you can read the docstring for the ``preprocess`` and ``make_caller`` methods below.